Formula/powershell-lts.rb

```diff
@@ -29,8 +29,8 @@ class PowershellLts < Formula
   version "7.4.11"
   version_scheme 1
 
-  # .NET Core 3.1 requires High Sierra - https://docs.microsoft.com/en-us/dotnet/core/install/dependencies?pivots=os-macos&tabs=netcore31
-  depends_on macos: :high_sierra
+  # .NET 8 requires macOS 13 (Ventura) - https://github.com/dotnet/core/blob/main/release-notes/8.0/supported-os.md#apple
+  depends_on macos: :ventura
```

Comment:
```suggestion
  depends_on macos: :catalina
```

Another option: .NET 8 was originally released with support for macOS 10.15 (Catalina)